apps/root/src/common/utils/transaction-history/parsing.tsx

```diff
@@ -885,7 +910,8 @@ export const transformNonIndexedEvents = ({
         baseEventData = buildBaseDcaPendingEventData(position);
 
         const totalBefore = position.rate.amount * BigInt(position.remainingSwaps);
-        const totalNow = BigInt(event.typeData.newRate) * BigInt(event.typeData.newSwaps);
+        const newRate = parseUnits(event.typeData.newRate, event.typeData.decimals);
```

Comment:
were we actually receiving the value as "0.1" ?

Reply:
Yep. After thinking about this, I finally parsed this in the component 😄